KIPs/kip-7-fungible_token.md

```diff
@@ -170,7 +211,39 @@ OPTIONAL - This method can be used to improve usability,
 but interfaces and other contracts MUST NOT expect these values to be present.
 
 ```solidity
-function mint(address _to, uint256 _value) public returns (bool) {
+function mint(address _to, uint256 _value) public returns (bool)
```

Comment:
Please add description above something like:

The function MUST `throw` if the sender is not a minter.

Reply:
MUST? SHOULD? What do you prefer?

Reply:
For `throw`, `SHOULD` is used commonly.

Reply:
I added the description.